x/evm/precompile/test/export.go

```diff
@@ -31,68 +28,52 @@ func SetupWasmContracts(deps *evmtest.TestDeps, s *suite.Suite) (
 ) {
 	wasmCodes := DeployWasmBytecode(s, deps.Ctx, deps.Sender.NibiruAddr, deps.App)
 
-	otherArgs := []struct {
-		InstMsg []byte
-		Label   string
+	instantiateArgs := []struct {
+		InstantiateMsg []byte
+		Label          string
 	}{
 		{
-			InstMsg: []byte("{}"),
-			Label:   "https://github.com/NibiruChain/nibiru-wasm/blob/main/contracts/nibi-stargate/src/contract.rs",
+			InstantiateMsg: []byte("{}"),
+			Label:          "https://github.com/NibiruChain/nibiru-wasm/blob/main/contracts/nibi-stargate/src/contract.rs",
 		},
 		{
-			InstMsg: []byte(`{"count": 0}`),
-			Label:   "https://github.com/NibiruChain/nibiru-wasm/tree/ec3ab9f09587a11fbdfbd4021c7617eca3912044/contracts/00-hello-world-counter",
+			InstantiateMsg: []byte(`{"count": 0}`),
+			Label:          "https://github.com/NibiruChain/nibiru-wasm/tree/ec3ab9f09587a11fbdfbd4021c7617eca3912044/contracts/00-hello-world-counter",
 		},
 	}
 
-	for wasmCodeIdx, wasmCode := range wasmCodes {
+	for i, wasmCode := range wasmCodes {
 		s.T().Logf("Instantiate using Wasm precompile: %s", wasmCode.binPath)
 		codeId := wasmCode.codeId
 
 		m := wasm.MsgInstantiateContract{
 			Admin:  "",
 			CodeID: codeId,
-			Label:  otherArgs[wasmCodeIdx].Label,
-			Msg:    otherArgs[wasmCodeIdx].InstMsg,
-			Funds:  []sdk.Coin{},
+			Label:  instantiateArgs[i].Label,
+			Msg:    instantiateArgs[i].InstantiateMsg,
 		}
 
 		msgArgsBz, err := json.Marshal(m.Msg)
 		s.NoError(err)
 
-		var funds []precompile.WasmBankCoin
-		fundsJson, err := m.Funds.MarshalJSON()
-		s.NoErrorf(err, "fundsJson: %s", fundsJson)
-		err = json.Unmarshal(fundsJson, &funds)
-		s.Require().NoError(err)
-
-		callArgs := []any{m.Admin, m.CodeID, msgArgsBz, m.Label, funds}
+		callArgs := []interface{}{m.Admin, m.CodeID, msgArgsBz, m.Label, []precompile.WasmBankCoin{}}
```

Comment:
I think `any` is preferred over `interface{}` in modern Go. It's also more concise